Formula/rizin.rb

```diff
@@ -0,0 +1,56 @@
+class Rizin < Formula
+  desc "UNIX-like reverse engineering framework and command-line toolset"
+  homepage "https://rizin.re"
+  url "https://github.com/rizinorg/rizin/releases/download/v0.3.1/rizin-src-v0.3.1.tar.xz"
+  sha256 "9ae4b7f9c47be63665fd0e59375b2b10f83c37156abb044ca4d61c1f0fc88f7e"
+  license "LGPL-3.0-only"
+  head "https://github.com/rizinorg/rizin.git", branch: "dev"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "capstone"
+  depends_on "libuv"
+  depends_on "libzip"
+  depends_on "lz4"
+  depends_on "openssl@3"
+  depends_on "tree-sitter"
+  depends_on "xxhash"
+
+  uses_from_macos "zlib"
+
+  def install
+    (buildpath/"pkgconfig").install_symlink Formula["xxhash"].opt_lib/"pkgconfig/libxxhash.pc" => "xxhash.pc"
+    ENV.prepend_path "PKG_CONFIG_PATH", buildpath/"pkgconfig"
+
+    mkdir "build" do
+      args = [
+        "-Dpackager=#{tap.user}",
+        "-Dpackager_version=#{pkg_version}",
+        "-Duse_sys_libzip=enabled",
+        "-Duse_sys_zlib=enabled",
+        "-Duse_sys_lz4=enabled",
+        "-Duse_sys_tree_sitter=enabled",
+        "-Duse_sys_libuv=enabled",
+        "-Duse_sys_openssl=enabled",
+        "-Duse_sys_capstone=enabled",
+        "-Duse_sys_xxhash=enabled",
+        "-Drizin_plugins=${HOMEBREW_PREFIX}/lib/rizin/rizin-plugins",
+        "-Drizin_extras=${HOMEBREW_PREFIX}/lib/rizin/rizin-extra",
+        "-Drizin_bindings=${HOMEBREW_PREFIX}/lib/rizin/rizin-bindings",
```

Comment:
```suggestion
        "-Drizin_plugins=${HOMEBREW_PREFIX}/lib/rizin/plugins",
        "-Drizin_extras=${HOMEBREW_PREFIX}/lib/rizin/extras",
        "-Drizin_bindings=${HOMEBREW_PREFIX}/lib/rizin/bindings",
```
The extra `rizin-` seems superfluous, no? I also wonder if we should have caveats indicating these custom locations.

Reply:
I updated paths and added a caveat